bash/01_dataloading.sh

```diff
@@ -40,7 +40,7 @@ case $MODE in
     *) 
         import_public dob_permitissuance $DOB_DATA_DATE &
         import_public dob_jobapplications $DOB_DATA_DATE &
-        import_public dob_geocode_results &
+        import_public dob_geocode_results $DOB_DATA_DATE &
```

Comment:
I think we need to have some better documentation about this dataloading step

Reply:
lol totally. Also it is about the data sync step needs to documentation that if you run workflow dispatch. Do not leave the date as latest which is the default